server/manifest.go

```diff
@@ -57,6 +58,18 @@ func (m *Manifest) RemoveLayers() error {
 	return nil
 }
 
+func ValidateManifest(m *Manifest, path string) error {
```

Comment:
```suggestion
func validateManifest(m *Manifest, path string) error {
```

Lowercase should so since we don't have to expose this function

Reply:
Two more small comments:

* Do we want to make this a function of the `Manifest` (e.g. `m.Valid()`)?
* Is the `path` parameter here required?

Reply:
I changed Validate to be a function of Manifest and also moved path to be handled by the caller. Given the first change, I kept the other two items as they were to be consistent with the other components of Manifest.

I also fixed the lint and test failures.